hail/python/hail/table.py

```diff
@@ -405,13 +413,13 @@ def _force_count(self):
     def _select(self, caller, row):
         analyze(caller, row, self._row_indices)
         base, cleanup = self._process_joins(row)
-        return cleanup(Table(base._jt.mapRows(str(row._ir))))
+        return cleanup(Table(TableMapRows(base._tir, row._ir)))
```

Comment:
Curious: can you get rid of Scala Table.mapRows now?

Reply:
Not yet. Maybe in a future PR.

Reply:
Because other things are calling it?

Reply:
Yes, `key_by` and something with a join.